### PR TITLE
Feature + Fix: Added new chat filters, Fixed: Healer Orb Pickup

### PR DESCRIPTION
**Pull request by Mikecraft1224**
## What
Added chat filters for rare drops from chests, sacrifice and fixed healer orb pickups

## Changelog New Features
+ Added new chat filters. - Mikecraft1224
    * Filters for rare dungeon chest rewards and sacrifice messages from other players.

## Changelog Fixes
+ Fixed healer orb pickup hider. - Mikecraft1224

### CODE REVIEW

**Review by CalMWolfs — APPROVED** [review]
code lgtm

File: src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt [hunk 1]
```diff
@@ -354,6 +354,10 @@ class ChatFilter {
         "§7Your §r§aRabbit Barn §r§7capacity has been increased to §r§a.* Rabbits§r§7!".toPattern(),
         "§7You will now produce §r§6.* Chocolate §r§7per click!".toPattern()
     )
+    private val sacrificePatterns = listOf(
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt:362]: please insert regex-test cases

File: src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt [hunk 2]
```diff
@@ -371,6 +375,10 @@ class ChatFilter {
         "▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬",
     )
 
+    private val rareDropsMessages = listOf(
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt:386]: please insert regex-test cases

**Review by hannibal002 — DISMISSED** [review]
Add regex-test cases

**Review by CalMWolfs — APPROVED** [review]

**Pull request merged**